src/modules/musig/tests_impl.h

```diff
@@ -857,22 +873,35 @@ void musig_tweak_test(secp256k1_scratch_space *scratch) {
     /* Compute P0 = keyagg(pk0, pk1) and test signing for it */
     CHECK(secp256k1_musig_pubkey_agg(ctx, scratch, &P_xonly[0], &keyagg_cache, pk_ptr, 2) == 1);
     musig_tweak_test_helper(&P_xonly[0], sk[0], sk[1], &keyagg_cache);
+    CHECK(secp256k1_musig_pubkey_get(ctx, &P[0], &keyagg_cache));
 
-    /* Compute Pi = |Pj| + tweaki*G where where j = i-1 and try signing for
-     * that key. The function |.| normalizes the point to have an even
-     * X-coordinate. This results in ordinary "xonly-tweaking". */
+    /* Compute Pi = f(Pj) + tweaki*G where where j = i-1 and try signing for
+     * that key. If xonly is set to true, the function f is normalizes the input
+     * point to have an even X-coordinate (ordinary "xonly-tweaking").
```

Comment:
Since there's already "ordinary" vs. "xonly" tweaking, would it be more clear to just say "xonly-tweaking" here?

Reply:
Yeah that's better